{"uid":"ac2c218f7fe072d3","name":"test_validate_payment[api-19]","fullName":"tests.endpoints.test_payments#test_validate_payment","historyId":"5dad880742ddfd39d1015f29f76581bb","time":{"start":1733072127503,"stop":1733072128382,"duration":879},"status":"broken","statusMessage":"pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nusers\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing\nmeta\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing","statusTrace":"api_client = <api_handler.ApiClient.ApiClient object at 0x7fa2d9665610>\nx_task_value = 'api-19'\n\n    @allure.feature(\"Orders API\")\n    @allure.story(\"Duplicate item in order\")\n    @pytest.mark.parametrize(\"x_task_value\", [\"api-19\"])\n    def test_validate_payment(api_client, x_task_value):\n        users_response = api_client.get_users(x_task_value, 0)\n>       users_dto = Users(**users_response.json())\nE       pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nE       users\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\nE       meta\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\n\ntests/endpoints/test_payments.py:14: ValidationError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1733072114640,"stop":1733072114642,"duration":2},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"base_url","time":{"start":1733072114642,"stop":1733072114642,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"api_client","time":{"start":1733072114642,"stop":1733072114642,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"setup_test","time":{"start":1733072114643,"stop":1733072117360,"duration":2717},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"worker_id","time":{"start":1733072114643,"stop":1733072114643,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"broken","statusMessage":"pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nusers\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing\nmeta\n  Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.9/v/missing","statusTrace":"api_client = <api_handler.ApiClient.ApiClient object at 0x7fa2d9665610>\nx_task_value = 'api-19'\n\n    @allure.feature(\"Orders API\")\n    @allure.story(\"Duplicate item in order\")\n    @pytest.mark.parametrize(\"x_task_value\", [\"api-19\"])\n    def test_validate_payment(api_client, x_task_value):\n        users_response = api_client.get_users(x_task_value, 0)\n>       users_dto = Users(**users_response.json())\nE       pydantic_core._pydantic_core.ValidationError: 2 validation errors for Users\nE       users\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\nE       meta\nE         Field required [type=missing, input_value={'code': 500, 'message': 'Internal server error'}, input_type=dict]\nE           For further information visit https://errors.pydantic.dev/2.9/v/missing\n\ntests/endpoints/test_payments.py:14: ValidationError","steps":[],"attachments":[{"uid":"40f56fda4be0aba7","name":"log","source":"40f56fda4be0aba7.txt","type":"text/plain","size":318},{"uid":"427daffdad864335","name":"stderr","source":"427daffdad864335.txt","type":"text/plain","size":314}],"parameters":[],"stepsCount":0,"attachmentsCount":2,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"setup_test::0","time":{"start":1733072128386,"stop":1733072128386,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"feature","value":"Orders API"},{"name":"story","value":"Duplicate item in order"},{"name":"parentSuite","value":"tests.endpoints"},{"name":"suite","value":"test_payments"},{"name":"host","value":"fv-az1249-467"},{"name":"thread","value":"2339-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.endpoints.test_payments"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"x_task_value","value":"'api-19'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"ac2c218f7fe072d3.json","parameterValues":["'api-19'"]}